### PR TITLE
lua_api.txt: author, release and title fields when downloading from ContentDB

### PR DESCRIPTION
**Pull request by Zughy**
Fixes https://github.com/minetest/minetest/issues/8988

## To do

This PR is Ready for Review.

### CODE REVIEW

**Comment by Zughy** [comment]
Guys, c'mon, 23 days to check 9 lines of text...

File: doc/lua_api.txt [hunk 1]
```diff
@@ -79,6 +79,12 @@ The game directory can contain the following files:
       e.g. `disallowed_mapgen_settings = mgv5_spflags`
       These settings are hidden for this game in the world creation
       dialog and game start menu.
+    * `author`: The author of the game. It only appears when downloaded from
+                ContentDB.
+    * `release`: The ID corresponding to the version of the mod. It only
```
Comment (rubenwardy) [inline doc/lua_api.txt:84]: This should never be set by modders

File: doc/lua_api.txt [hunk 2]
```diff
@@ -79,6 +79,12 @@ The game directory can contain the following files:
...
+                 appears when downloaded from ContentDB.
+    * `title`: The title set on ContentDB. It only appears when downloaded
```
Comment (rubenwardy) [inline doc/lua_api.txt:86]: not necessarily a CDB only thing

File: doc/lua_api.txt (hunk 1 above)
Comment (rubenwardy) [reply doc/lua_api.txt:84]: "Should only ever be set by ContentDB, as it is an internal ID used to track versions`

File: doc/lua_api.txt [hunk 3]
```diff
@@ -62,12 +62,13 @@ Where `<gameid>` is unique to each game.
 The game directory can contain the following files:
 
 * `game.conf`, with the following keys:
-    * `name`: Required, human readable name  e.g. `name = Minetest`
+    * `name`: Required. Allows Minetest to determine the game name even if the
```
Comment (rubenwardy) [inline doc/lua_api.txt:65]: this is not correct - `name` is the same as a mod's `title` for games. Games don't have an equivalent for `name` yet

File: doc/lua_api.txt [hunk 4]
```diff
@@ -79,6 +80,11 @@ The game directory can contain the following files:
       e.g. `disallowed_mapgen_settings = mgv5_spflags`
       These settings are hidden for this game in the world creation
       dialog and game start menu.
+    * `author`: The author of the game. It only appears when downloaded from
+                ContentDB.
+    * `release`: Ignore this: Should only ever be set by ContentDB, as it is
+                 an internal ID used to track versions.
+    * `title`: A human-readable title to address the game.
```
Comment (rubenwardy) [inline doc/lua_api.txt:87]: hmm, I should confirm whether this does anything

Comment (Zughy) [reply doc/lua_api.txt]: Actually it doesn't appear at all when downloading from CDB, my bad

File: doc/lua_api.txt [hunk 5]
```diff
@@ -62,12 +62,12 @@ Where `<gameid>` is unique to each game.
 The game directory can contain the following files:
 
 * `game.conf`, with the following keys:
-    * `name`: Required, human readable name  e.g. `name = Minetest`
+    * `name`: Required, human readable name  e.g. `name = Minetest`.
```
Comment (rubenwardy) [inline doc/lua_api.txt:65]:
```suggestion
    * `name`: Required, a human readable title to address the game, e.g. `name = Minetest`.
```

**Review by rubenwardy — APPROVED** [review]

**Comment by Zughy** [comment]
Bump, this is probably the fastest PR to check (and approve) in here

**Comment by paramat** [comment]
:+1:

**Pull request merged**